system/Common.php

```diff
@@ -563,6 +563,27 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('htmltag_endline')) {
+    /**
+     * Endline HTML Tag
+     *
+     * Generates endline html tag by document type
+     *
+     * @internal
+     */
+    function htmltag_endline(): string
```

Comment:
I think this method name is a little different.
Native English speakers, please give us a more appropriate name.

Reply:
What about `selfClosingHTMLTag` or `selfclosing_html`?

Ref : https://validator.w3.org/check

Reply:
`>` or `/>` are not the self-closing tags referred by the HTML standard. This is just the [`U+003E GREATER-THAN SIGN character (>)`](https://html.spec.whatwg.org/multipage/syntax.html#start-tags)

`<html></html>` => standard elements with start and end tags
`<input>` => void element with start tag only
`<path/>` => foreign element in SVG namespace that is self-closing

Reply:
Thank you @paulbalandan 
Then `html_tag_greater_than_sign()`?

Reply:
or `greater_than_sign_element`?

Reply:
Since this is just an internal function whether or not to include the solidus (`/`) in the `>` output, why not make this function output the solidus only?
```php
if (! function_exists('solidus')) {
    function solidus(): string
    {
        if (config('DocTypes')->xhtml ?? false) {
            return '/';
        }

        return '';
    }
}
```

Reply:
The implementation seems better!

But it does not seem good that using global function name for an internal function.
If a dev has the same name function, it breaks the existing app.

Why don't you use static class method?
E.g., `CodeIgniter\Helper\FormHelper::solidus()`

Reply:
I agree. It pollutes too much the global namespace.

Maybe, if [this FR](https://forum.codeigniter.com/showthread.php?tid=80904) became acceptable, this should go with it.